gateway/api_test.go

```diff
@@ -2518,14 +2435,7 @@ func TestOAS(t *testing.T) {
 
 			expectedTykExt.Server.ListenPath.Value = listenPath
 			expectedTykExt.Upstream.URL = upstreamURL
-			expectedTykExt.Server.CustomDomain = &oas.Domain{
-				Enabled: true,
-				Name:    customDomain,
-			}
-			expectedTykExt.Server.GatewayTags = &oas.GatewayTags{
```

Comment:
Removed in PR but, styling wise:

- tags should be defined multi-line, so they are friendly to read (one field per line)
- tags should always define full name keys e.g. none of {true, customDomain} (not the case here)

Ideally we'd run a LSC change that expands all structs in the codebase... one day.

Reply:
Think of it calling a constructor function. When it has short amount of fields to set, it is ok to set in one line like a constructor.